Adds sensory.dm restoration and destruction viruses

LanCartwright opened the pull request:
![image](https://cloud.githubusercontent.com/assets/13142400/12957493/d4a6c13c-d024-11e5-8637-b3f83492cbed.png)

:cl:
:rscadd:Adds Sensory restoration and destruction. Effects are it's titular namesake. Both are level 6 symptoms.
/:cl:

LanCartwright committed:
Adds sensory.dm restoration and destruction viruses

Buggy123 commented:
:+1:

PKPenguin321 commented:
should probably eliminate ocular restoration/blindness symptoms and merge them with this

kevinz000 commented:
seems overpowered.

PKPenguin321 commented:
Eh, ear/brain damage and hallucinations aren't exactly that common. Hardly overpowered.

The one that makes you take damage to almost all senses, on the other hand, might be a bit much.

LanCartwright commented:
I'm highly considering removing deafness from this, due to the fact that deafness.dm does actually exist.

I'm unsure what to replace it with.

KorPhaeron commented:
Do me a favour for all of these viruses:

Load up a local server, infect yourself, then time how long until you are incapacitated.

GunHog commented:
## Test EVERYTHING!!!!

LanCartwright commented:
oh right the constant 10u of mindbreaker
fucking kek i'll change this

LanCartwright committed:
nerfs sensory destruction effects

LanCartwright committed:
Testing time to kill sensory.dm

LanCartwright committed:
THIS IS TESTED AND BALANCED.
Fixes all sensory.dm related issues.
Adds presets.dm for ease of administrative authorized or unauthorized use during a round.

RemieRichards commented on code/datums/diseases/advance/symptoms/sensory.dm:
```diff
@@ -0,0 +1,95 @@
+			if(2)
+				M << "<span class='warning'><b>You can't anything with your fingers.</b></span>"
+			if(3)
+				M.reagents.add_reagent("ethanol"=rand(7,25))
```
"ethanol", rand(7,25)
not =

GunHog commented:
# Testing

LanCartwright committed:
replaces some  = with ,

tkdrg commented on code/datums/diseases/advance/symptoms/sensory.dm:
```diff
@@ -0,0 +1,95 @@
+			if(2)
+				if(M.reagents.get_reagent_amount("inacusiate")<10)
+					M.reagents.add_reagent("inacusiate"=10)
+					M << "<span class='notice'><b>Your hearing feels clearer and crisp</b></span>"
```
Missing the period at the end
Also the bold text in these looks weird and inconsistent imo

GunHog replied:
Agreed, should lose the bold text for the heal version.

tkdrg commented on code/datums/diseases/advance/symptoms/sensory.dm:
```diff
@@ -0,0 +1,95 @@
+	Lowers resistance tremendously.
+	Decreases stage speed tremendously.
+	Decreases transmittablity tremendously.
+	Fatal.
```
how is this fatal
i don't understand what you're trying to do with this

GunHog replied:
"Fatal" refers to the reagent level, not the threat of the virus itself.

tkdrg commented on code/datums/diseases/advance/symptoms/sensory.dm:
```diff
@@ -0,0 +1,95 @@
+					M << "<span class='notice'><b>You feel sober.</b></span>"
+			if(4)
+				if(M.reagents.get_reagent_amount("antihol") < 10 && M.reagents.get_reagent_amount("inacusiate") < 10 && M.reagents.get_reagent_amount("synaptizine") < 10)
+					M.reagents.add_reagent_list(list("antihol"=10, "inacusiate"=10, "synaptizine"=5))
```
not quite sure if synap is OP here, a constant pump of stun-reducing drugs sounds a bit strong

PKPenguin321 replied:
Stimulant tho?

tkdrg commented on code/datums/diseases/advance/symptoms/sensory.dm:
```diff
@@ -0,0 +1,95 @@
+			if(1)
+				M << "<span class='warning'><b>You can't taste a thing.</b></span>"
+			if(2)
+				M << "<span class='warning'><b>You can't anything with your fingers.</b></span>"
```
i cant what

GunHog replied:
It is a simulation of your reduced capability to form coherent thoughts.

LanCartwright commented:
Fixed grammar. Still need to debate on synaptizine, and maybe if I should adjust it to use a special kind of synaptizine.

LanCartwright committed:
fixes typos, adjusts some stuff
